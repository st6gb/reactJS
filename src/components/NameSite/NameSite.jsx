import React from 'react';
import classNames from 'classnames/bind';
import styles from './namesite.scss';

const cx = classNames.bind(styles);

function NameSite(){
  return <div className={cx('content')}>nextflixroulette</div>
}

export default NameSite;
