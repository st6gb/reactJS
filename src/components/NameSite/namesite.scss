.content{
  color: rgb(250, 28, 65);
  font-size: 150%;
  padding: 10px 0 10px 0
}