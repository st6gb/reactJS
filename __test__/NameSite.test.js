import React from 'react';
import { shallow } from 'enzyme';
import  NameSite  from '../src/components/NameSite/NameSite.jsx';

describe('Name site', ()=>{

  it('render namesite', ()=>{
    const nameSite = shallow(<NameSite />);
    expect(nameSite).toMatchSnapshot();
  });
})