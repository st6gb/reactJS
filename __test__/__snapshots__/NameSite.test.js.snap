// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <NameSite />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": "nextflixroulette",
      "className": "content",
    },
    "ref": null,
    "rendered": "nextflixroulette",
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "nextflixroulette",
        "className": "content",
      },
      "ref": null,
      "rendered": "nextflixroulette",
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;

exports[`Name site render namesite 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <NameSite />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": "nextflixroulette",
      "className": "content",
    },
    "ref": null,
    "rendered": "nextflixroulette",
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "nextflixroulette",
        "className": "content",
      },
      "ref": null,
      "rendered": "nextflixroulette",
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
